src/imgui_impl_sdl_gl3.cc

```diff
@@ -426,6 +428,9 @@ void ImGui_ImplSdlGL3_NewFrame(SDL_Window* window)
 {
     ImGuiIO& io = ImGui::GetIO();
 
+    if (!io.Fonts->TexID)
+        ImGui_ImplSdlGL3_CreateDeviceObjects();
```

Comment:
Are we better off doing this at context creation rather than lazily later?

Reply:
Possibly. Would mean another impl wrapper.